kore/src/Kore/Repl.hs

```diff
@@ -273,6 +277,6 @@ runRepl axioms' claims' logger replScript replMode outputFile = do
     prompt = do
         node <- Lens.use (field @"node")
         liftIO $ do
-            putStr $ "Kore (" <> show (unReplNode node) <> ")> "
+            putStr $ "[kore-repl] Kore (" <> show (unReplNode node) <> ")> "
```

Comment:
I don't think `[kore-repl]` is necessary here, this isn't an error.

Reply:
Oh, I just realized "terminal message" doesn't mean "message printed to terminal"